vespalib/src/vespa/vespalib/datastore/sharded_hash_map.h

```diff
@@ -41,7 +41,7 @@ class ShardedHashMap {
     std::atomic<FixedSizeHashMap *> _maps[num_shards];
     std::unique_ptr<const EntryComparator> _comp;
 
-    size_t get_shard_idx(const EntryComparator& comp, EntryRef key_ref) const;
+    VESPA_DLL_LOCAL static size_t get_shard_idx(const EntryComparator& comp, EntryRef key_ref) const;
```

Comment:
Consider removing const (due to addition of static).

Reply:
Fixed, also needed string.h to get VESPA_DLL_LOCAL.
However there is a followup PR to change that soon to come.